consumer/handler/func.go

```diff
@@ -0,0 +1,20 @@
+package handler
+
+import (
+	"github.com/heetch/felice/message"
+)
+
+// A HandleMessageFn is a function type that can be passed to
+// HandlerFunc.  Its signature mimics the interface of the
+// HandleMessage function in the Handler interface.
+type HandleMessageFn func(*message.Message) error
+
+func (h *HandleMessageFn) HandleMessage(msg *message.Message) error {
+	return (*h)(msg)
+}
+
+// HandlerFunc wraps a HandleMessagFn in such a way that it complies
+// with the Handler interface.
+func HandlerFunc(fn HandleMessageFn) Handler {
```

Comment:
🌷 You don't need that function actually, a cast does the same thing. You can rename `HandleMessageFn` to `HandlerFunc` and let the user cast his func to a `HandlerFunc` (cf https://golang.org/src/net/http/server.go?s=59707:59754#L1950)

Reply:
Yes! Nice.  :champagne: